clients/cloud/go/ccloud_lib/ccloud_lib.go

```diff
@@ -0,0 +1,76 @@
+package ccloud_lib
+
+/**
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+    "os"
+    "flag"
+    "bufio"
+    "strings"
+    "fmt"
+)
+
+type RecordValue struct {
+    Count   int
+}
+
+func ParseArgs() (*string, *string) {
+
+    config_file := flag.String("f", "", "path to Confluent Cloud configuration file")
+    topic := flag.String("t", "", "topic name")
+    flag.Parse()
+    if *config_file == "" || *topic == "" {
+        flag.Usage()
+        os.Exit(2) // the same exit code flag.Parse uses
+    }
+
+    return config_file, topic
+
+}
+
+
+func ReadCCloudConfig(config_file string) (map[string]string) {
+
+    m := make(map[string]string)
```

Comment:
@edenhill this is intentional.  If there are garbage fields in the config file, then it will fail.  This simple example should not be validating the config file, it should cherry-pick absolute minimum to ensure success.